src/lib/localStorage.ts

```diff
@@ -8,6 +8,7 @@ export const setLocalStorage = <Value>({ key, value }: { key: LocalStorageKey; v
   const serializedValue = JSON.stringify(value);
 
   globalThis.localStorage?.setItem(key, serializedValue);
+  return undefined;
```

Comment:
instead of returning `undefined` can this fn be re-written?

```
if (value === undefined) {
  removeLocalStorage({ key });
  return;
}
```

Reply:
yep